config/crds-patches/apm-and-kibana-podtemplate-patch.yaml

```diff
@@ -0,0 +1,11 @@
+# Using `kubectl apply` stores the complete CRD file as an annotation,
+# which may be too big for the annotations size limit.
+# One way to mitigate this problem is to remove the (huge) podTemplate properties from the CRD.
+# It also avoids the problem of having any k8s-version specific field in the Pod schema,
+# that would maybe not match the user's k8s version.
+- op: remove
+  path: /spec/validation/openAPIV3Schema/properties/spec/properties/podTemplate/properties
+# Remove validation.Type that causes failures on k8s 1.11.
+# TODO: that should already not be there https://github.com/kubernetes-sigs/controller-tools/pull/72
+- op: remove
+  path: /spec/validation/openAPIV3Schema/type
```

Comment:
🤷‍♂ looks like a regression on controller-gen

Reply:
Maybe add "TODO" here so we can grep for it later once it's been fixed (or open an issue?)

Reply:
there is a TODO already?